components/Slider/slider.css

```diff
@@ -0,0 +1,34 @@
+.slider-container {
+  width: 100%;
+}
+
+.slider {
+  -webkit-appearance: none;
+  width: 100%;
+  height: 15px;
+  background: #d3d3d3;
```

Comment:
I think you missed using color variable here

Reply:
Yes right, thank you. I have updated now.